dialects/hsql/src/main/resources/META-INF/services/app.cash.sqldelight.dialect.api.SqlDelightDialect

```diff
@@ -0,0 +1 @@
+app.cash.sqldelight.dialects.hsql.HsqlDialect
```

Comment:
You can use auto-service to generate this file automatically if you want. You just do `@AutoService(SqlDelightDialect::class)` on `HsqlDialect` and never have to think about maintaining the correct name/location of this file or its contents across refactors.

Same applies to all dialects below.

Reply:
I opted not to so I wasn't introducing kapt. Not that it would do a ton, just felt like that was more than was necessary here